spec/factories/tags.rb

```diff
@@ -1,5 +1,5 @@
 FactoryBot.define do
   factory :tag do
-    sequence(:name){ |n| "tag-#{n}" }
+    name { "!#{Random.bytes(3).unpack1('H*')}_tag" }
```

Comment:
this seems like a much more expensive way to build a name, I'm curious about the thinking behind it?

Reply:
My previous implementation was `sequence(:name){ |n| "!00000#{n}"tag }` which was not generating the correct format of hex code and hence specs were failing. Therefore I have moved to `Random.bytes(3).unpack1('H*')` which will surely create a unique and ensures spec failure for uniqueness. I will be looking for an inexpensive solution.

Reply:
I have found an alternative solution `sequence(:name) { |n| "!#{"%06d" % n}_tag" }` to generate hex code, but we have a limitation for this we can generate hex code from `000000 to 999999` after that we will get failed specs because the hex code will exceed 7 digits and will get a validation error. But I do not think we have to worry about that limitation because we won't be generating up to 999999 tags in RSpec. Thoughts?